test/extended/tbr_health/check.go

```diff
@@ -0,0 +1,16 @@
+package tbr_health
+
+import (
+	g "github.com/onsi/ginkgo"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-devex][Suite:openshift/conformance/parallel] check registry.redhat.io is available and samples operator can import sample imagestreams", func() {
```

Comment:
```suggestion
var _ = g.Describe("[sig-devex] check registry.redhat.io is available and samples operator can import sample imagestreams", func() {
```

the conformance bit gets added automatically

Reply:
interesting

after I see that it is included as is, I'll pull in this commit and we'll go from there

thanks

Reply:
given I have to change anyway to check for error I'll pull this in @bparees